user/languages/objective-c.md

```diff
@@ -33,11 +33,97 @@ NDA imposed on them. We do test them internally, and our goal is to make new
 versions available the same day they come out. If you have any further questions
 about Xcode pre-release availability, send us an email at support@travis-ci.com._
 
+### Xcode 7.2 Beta
+
+Xcode 7.2 Beta is available by adding `osx_image: xcode7.2b1` to your .travis.yml.
+
```

Comment:
Can you add a **bold** note like **Note: This is our first build image which is running on 10.11, as well as having the Xcode beta** ?